src/Classes/FilterButtonsDisplayer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Fetch\Classes;
+
+class FilterButtonsDisplayer {
+
+    public static function displayFilterButtons(array $temperamentsArray, string $currentTemperament = ''): string
+    {
+        $result = '';
+        foreach ($temperamentsArray as $temperament) {
+            if(gettype($temperament) === 'string') {
+                $result .= '<a href="index.php" class="chip';
```

Comment:
add tabindex="2" for accessibility tabbing and add aria-label filter selection by blank temperament